blocks/procedures.js

```diff
@@ -59,6 +59,7 @@ Blockly.Blocks['procedures_defnoreturn'] = {
     this.setTooltip(Blockly.Msg.PROCEDURES_DEFNORETURN_TOOLTIP);
     this.setHelpUrl(Blockly.Msg.PROCEDURES_DEFNORETURN_HELPURL);
     this.arguments_ = [];
+    this.argumentVarModels_ = [];
```

Comment:
Since argument names can be derived from the argumentVarModels_, is there a point to keep parallel lists?

Reply:
I think I can get rid of the arguments_ list.  I didn't yet because it would mean reaching into more code and expanding the scope of this (already large) PR.  I'll make an issue to clean that up.

Reply:
https://github.com/google/blockly/issues/1532